src/marqo/core/distributed_lock/distributed_lock_manager.py

```diff
@@ -0,0 +1,17 @@
+from marqo.vespa.marqo_zookeeper_client import MarqoZookeeperClient
+from marqo.core.distributed_lock.zookeeper_distributed_lock import ZookeeperDistributedLock
+
+_DEPLOYMENT_LOCK_PATH = "/marqo__deployment_lock"
+
+
+def get_deployment_lock(zookeeper_client: MarqoZookeeperClient, acquire_timeout=1) -> ZookeeperDistributedLock:
```

Comment:
Minor, but simply `ZookeeperClient` is more consistent with `VespaClient`

Reply:
done